src/docfx/docset/Context.cs

```diff
@@ -112,5 +121,18 @@ public void Delete(string destRelativePath)
                 File.Delete(destinationPath);
             }
         }
+
+        private sealed class Cache
```

Comment:
lift this class to a separate file.

Reply:
Keep it private for now since this is only used in Context.cs

---
In reply to: [223252803](https://github.com/dotnet/docfx/pull/3458#discussion_r223252803) [](ancestors = 223252803)